vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -494,6 +494,10 @@ supported:
               <PropertyValue Property="Value" String="SearchExpression" />
               <Annotation Term="Core.Description" String="String property can be used as first operand in `startswith`, `endswith`, and `contains` clauses" />
             </Record>
+            <Record>
+              <PropertyValue Property="Value" String="MultiValueOrSearchExpression" />
+              <Annotation Term="Core.Description" String="String property can be used like in `MultiValue` and like in `SearchExpression`" />
+            </Record>

```

Comment:
That's significantly shorter than the Description and LongDescription of MultiRangeOrSearchExpression.

Could you please align the two?